thefuck/output_readers/rerun.py

```diff
@@ -1,5 +1,6 @@
 import os
 import shlex
+import six
```

Comment:
Is this a new dependency? Presumably requirements.txt and the like will also need updating?